dashboard/test/models/script_test.rb

```diff
@@ -1006,9 +1006,77 @@ class CanViewVersion < ActiveSupport::TestCase
     assert_equal 'foo-2017', versions[1][:name]
     assert_equal '2017', versions[1][:version_year]
     assert_equal '2017', versions[1][:version_title]
+
+    course_versions = foo17.summarize(false, create(:teacher))[:course_versions]
+    assert_equal 2, course_versions.keys.length
+    assert_equal 'foo-2017', course_versions.values[0][:name]
+    assert_equal '2017', course_versions.values[0][:version_year]
+    assert_equal 'foo-2018', course_versions.values[1][:name]
+    assert_equal '2018', course_versions.values[1][:version_year]
+  end
+
+  test 'summarize_course_versions' do
```

Comment:
please say what business logic you are testing here

Reply:
I'm asking in part because I don't understand from the method comments what this method is doing, and sometimes unit tests can be a way to document the behavior.

Reply:
updated